README.md

```diff
@@ -19,98 +19,125 @@ npm install mosip/tuvali#v0.3.7
 ```
 
 # API documentation
-Firstly, for establishing the secured connection over BLE the connection params which include `cid` and `Public key` needs to be exchanged between two devices. The exchange of parameters can be accomplished, but is not limited to, by using a QR code.
+Firstly, for establishing the secured connection over BLE the Verifier's URI needs to be exchanged between two devices. The exchange of URI can be accomplished, but is not limited to, by using a QR code.
 
-For example use QR code generator to visually display params and QR code scanner to get params. A mobile app that displays a QR code can act as an `advertiser` by including its connection params as data in the QR code and another device can act as `discoverer` which scans the QR code, it can extract the connection params and initiate a BLE connection with the advertising device.
+For example use QR code generator to visually display URI and QR code scanner to read. A mobile app that displays a QR code can act as an `Verifier` by including its URI as data in the QR code and another device can act as `Wallet` which scans the QR code, it can extract the URI and initiate a BLE connection with the advertising device.
 
-> Note: The terms `advertiser` and `discoverer` are used for legacy API compatibility reasons. These terms will be replaced with `Verifier` and `Wallet` soon. Hence the terms are used interchangeably as required.
+## URI exchange and Establishing connection
 
-## Connection parameters exchange
-The device on which the QR code is displayed shall generate connection parameters using getConnectionParameters() method:
+### Verifier
+The Verifier device will be showing a QR code with URI. Verifier can generate URI and start advertisement using startAdvertisement() method. Once advertisement started, `Verifier` will keep on advertising URI and will wait for the `Wallet` to connect.
```

Comment:
The Verifier device can show a QR code with the URI. Verifier can generate URI through startAdvertisement() method. Once advertisement is started, `Verifier` will keep advertising with an advertisement payload derived from URI.

Reply:
updated